### PR TITLE
Changed characted and added a pic

### PR DESCRIPTION
**Pull request by floridop**
Added some content to your story. You are free to ask me to do any change you like before approving! Use this opportunity to experiment.

To complete this task you need to merge my contribution as explained in slides 84 to 89 keeping in mind that you are the release manager.

### CODE REVIEW

File: fairytale.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
 # MNXB11 fairytale
 
 Once upon a time,
-there was a **beautiful** _princess_
+there was a **ruthless** _hogre_
```
Comment (iMaxFlames) [inline fairytale.md]: Typo: Should be "ogre" not "hogre".

Comment (floridop) [reply fairytale.md]: Oops! sure! fixing

Comment (floridop) [reply fairytale.md]: Fixed in 21f6d5a55b853c63c140505e1c16bf3d68552593

**Comment by floridop** [comment]
I'll close this part of the homework, but if there is something else you're
curious to try and I am available I'll be happy to test.

**Pull request merged**